scripts/testing/install_dependencies.sh

```diff
@@ -36,16 +36,20 @@ TEMP=$(mktemp /tmp/anaconda.spec.XXXXXXX)
 # remove all problematic pieces from anaconda spec to be able to get dependencies
 sed 's/@PACKAGE_VERSION@/0/; s/@PACKAGE_RELEASE@/0/; s/%{__python3}/python3/' ./anaconda.spec.in > $TEMP
 
+# Ignore everything else than our dependencies from the spec files
+INSTALL_ONLY_DEPS="\
+python3- \
+-devel"
 # get all build requires dependencies from the spec file and strip out version
 # version could be problematic because of fedora version you are running and
 # they are mostly not important for automake
-build_deps=$(rpmspec -q --buildrequires $TEMP | sed 's/>=.*$//')
+build_deps=$(rpmspec -q --buildrequires $TEMP | grep -E "(${INSTALL_ONLY_DEPS// /|}| or )" | sed 's/>=.*$//')
```

Comment:
We need the build dependencies unconditionally otherwise the package wont compile.

Reply:
Good point. However, in that case we are also missing `git`. I'll add to the first list.